### PR TITLE
ci: add wildcard support to fly.io wsproxies

### PR DESCRIPTION
**Pull request by matifali**

### CODE REVIEW

File: .github/fly-wsproxies/paris-coder.toml [hunk 1]
```diff
@@ -2,16 +2,17 @@ app = "paris-coder"
 primary_region = "cdg"
 
 [experimental]
-  entrypoint = ["/opt/coder", "wsproxy", "server"]
+  entrypoint = ["CODER_DERP_SERVER_RELAY_URL=\"http://${FLY_PRIVATE_IP}:3000\"", "/opt/coder", "wsproxy", "server"]
```
Comment (deansheather) [inline .github/fly-wsproxies/paris-coder.toml:5]:
```suggestion
  entrypoint = ["/bin/sh", "-c", "CODER_DERP_SERVER_RELAY_URL=\"http://${FLY_PRIVATE_IP}:3000\" /opt/coder wsproxy server"]
```

File: .github/fly-wsproxies/sao-paulo-coder.toml [hunk 2]
```diff
@@ -21,6 +22,7 @@ primary_region = "gru"
   min_machines_running = 0
 
 [[vm]]
-  cpu_kind = "shared"
+  cpu_kind = "performance"
   cpus = 1
   memory_mb = 512
+
```
Comment (deansheather) [inline .github/fly-wsproxies/sao-paulo-coder.toml:28]:
```suggestion
```

File: .github/fly-wsproxies/paris-coder.toml [hunk 3]
```diff
@@ -21,6 +22,6 @@ primary_region = "cdg"
   min_machines_running = 0
```
Comment (deansheather) [inline .github/fly-wsproxies/paris-coder.toml:22]: 2 servers in paris

Comment (matifali) [reply .github/fly-wsproxies/paris-coder.toml:22]: I have set 2 in each using `fly scale count 2`
You can verify with `fly scale show`
This is for auto stopping when no one is using them

**Pull request merged**